src/Cli/dotnet/commands/dotnet-test/IPC/ObjectFieldIds.cs

```diff
@@ -9,55 +9,77 @@ namespace Microsoft.DotNet.Tools.Test
 {
     internal static class CommandLineOptionMessagesFieldsId
```

Comment:
I was thinking to something more like 

```cs
internal static class CommandLineOptionMessagesFieldsId
{
     public const int MessagesSerializerId = 3;
     .....
```

so all the info are in the same place for the same object type